### PR TITLE
Powrng 61

### PR DESCRIPTION
**Pull request by RafisSomeone**

### CODE REVIEW

File: src/main/java/com/uxtest/backend/dto/questionnaire/results/QuestionType.java [hunk 1]
```diff
@@ -9,4 +9,9 @@
     public String value() {
         return this.toString();
     }
+
+    public String getLowerCase(){
```
Comment (michalbarczyk) [inline src/main/java/com/uxtest/backend/dto/questionnaire/results/QuestionType.java:13]: method does more than its name suggeests

File: src/main/java/com/uxtest/backend/model/questionnaire/answer/LikertScaleAnswer.java [hunk 2]
```diff
@@ -32,4 +34,9 @@
     @ManyToOne
     @JoinColumn(nullable=false)
     private User user;
+
+    @Override
+    public AnswerExport getAnswerExport() {
+        return new AnswerExport(String.valueOf(answer),user);
```
Comment (michalbarczyk) [inline src/main/java/com/uxtest/backend/model/questionnaire/answer/LikertScaleAnswer.java:40]: add space before user

File: src/main/java/com/uxtest/backend/model/questionnaire/answer/MultipleChoiceAnswer.java [hunk 3]
```diff
@@ -34,4 +34,9 @@
     @ManyToOne
     @JoinColumn(nullable=false)
     private User user;
+
+    @Override
+    public AnswerExport getAnswerExport() {
+        return new AnswerExport(selectedOption.getContent(),user);
```
Comment (michalbarczyk) [inline src/main/java/com/uxtest/backend/model/questionnaire/answer/MultipleChoiceAnswer.java:40]: add space before user

File: src/main/java/com/uxtest/backend/model/questionnaire/answer/TextAnswer.java [hunk 4]
```diff
@@ -31,4 +33,9 @@
     @ManyToOne
     @JoinColumn(nullable=false)
     private User user;
+
+    @Override
+    public AnswerExport getAnswerExport() {
+        return new AnswerExport(answer,user);
```
Comment (michalbarczyk) [inline src/main/java/com/uxtest/backend/model/questionnaire/answer/TextAnswer.java:39]: add space before user

File: src/main/java/com/uxtest/backend/model/questionnaire/question/Question.java [hunk 5]
```diff
@@ -1,7 +1,24 @@
...
+
     ResultsDTO getResults();
+
+    List<? extends ExportDataAnswer> getDataAnswer();
+
+    QuestionType getQuestionType();
+
+    default QuestionExport getExportData(){
+        List<AnswerExport> answerExportList = getDataAnswer().stream().map(ExportDataAnswer::getAnswerExport).collect(toList());
+        return new QuestionExport(answerExportList, getContent(),getQuestionType());
```
Comment (michalbarczyk) [inline src/main/java/com/uxtest/backend/model/questionnaire/question/Question.java:22]: add space before getQuestionType()

File: src/main/java/com/uxtest/backend/model/questionnaire/answer/MultipleAnswerAnswer.java [hunk 6]
```diff
@@ -33,4 +31,15 @@
...
+
+    @Override
+    public AnswerExport getAnswerExport() {
+        StringBuilder builder = new StringBuilder();
+        for (MultipleAnswerQuestionOption option : selectedOptions){
+            builder.append(option.getContent());
+            builder.append(" ");
+        }
+        String answers = builder.toString();
+        return new AnswerExport(answers,user);
```
Comment (michalbarczyk) [inline src/main/java/com/uxtest/backend/model/questionnaire/answer/MultipleAnswerAnswer.java:43]: add space before user

File: src/main/java/com/uxtest/backend/dto/questionnaire/results/QuestionType.java (hunk 1 above)
Comment (RafisSomeone) [reply src/main/java/com/uxtest/backend/dto/questionnaire/results/QuestionType.java:13]: changed

File: src/main/java/com/uxtest/backend/model/questionnaire/answer/LikertScaleAnswer.java (hunk 2 above)
Comment (RafisSomeone) [reply src/main/java/com/uxtest/backend/model/questionnaire/answer/LikertScaleAnswer.java:40]: added

File: src/main/java/com/uxtest/backend/model/questionnaire/answer/MultipleChoiceAnswer.java (hunk 3 above)
Comment (RafisSomeone) [reply src/main/java/com/uxtest/backend/model/questionnaire/answer/MultipleChoiceAnswer.java:40]: added.

File: src/main/java/com/uxtest/backend/model/questionnaire/answer/TextAnswer.java (hunk 4 above)
Comment (RafisSomeone) [reply src/main/java/com/uxtest/backend/model/questionnaire/answer/TextAnswer.java:39]: added.

File: src/main/java/com/uxtest/backend/model/questionnaire/question/Question.java (hunk 5 above)
Comment (RafisSomeone) [reply src/main/java/com/uxtest/backend/model/questionnaire/question/Question.java:22]: added.

File: src/main/java/com/uxtest/backend/model/questionnaire/answer/MultipleAnswerAnswer.java (hunk 6 above)
Comment (RafisSomeone) [reply src/main/java/com/uxtest/backend/model/questionnaire/answer/MultipleAnswerAnswer.java:43]: added.

**Pull request merged**